packages/Checkboxes/src/index.tsx

```diff
@@ -2,41 +2,44 @@ import React, { Fragment, FunctionComponent, useEffect, useState } from "react";
 import uuidv1 from "uuid/v1";
 interface ICheckboxesProps {
   options?: any[] | object;
-  withEffect?: boolean;
-  bool?: boolean;
+  returnBoolean?: boolean;
   onChange: (...args: any) => void;
 }
 const Checkboxes: FunctionComponent<ICheckboxesProps> = ({
-  bool,
+  returnBoolean,
   onChange,
   options,
-  withEffect,
   ...attrs
 }): JSX.Element => {
-  const [data, setData]: any = useState(
-    Array.isArray(options) ? options : [options]
-  );
-  useEffect(() => {
-    if (withEffect) {
-      setData(options);
-    }
-  }, [options, withEffect]);
+  const formatedOptions = Array.isArray(options) ? options : [options];
+
+  const [data, setData]: any = useState(formatedOptions);
+
+  useEffect(() => setData(formatedOptions), [options]);
+
   const toggle = ({ event, item, key }: any): void => {
     if (item.disabled) {
       return;
     }
+
     data[key].checked = !item.checked;
     setData([...data]);
+
     let checked = data.filter((option: any): boolean => option.checked);
-    if (bool) {
+
+    if (returnBoolean) {
       checked = !!checked.length;
     }
+
     onChange({ event, checked, data });
   };
+
+  const requiredClassName: string = "required";
+
   return (
     <Fragment>
```

Comment:
Do you know that intead use <Fragment></Fragment> react understand also as fragment <> </> ;)

Reply:
the same++